src/tests/ftest/cat_recovery/ddb.py

```diff
@@ -0,0 +1,666 @@
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import ctypes
+import re
+from ClusterShell.NodeSet import NodeSet
+
+from apricot import TestWithServers
+from pydaos.raw import IORequest, DaosObjClass
+from general_utils import create_string_buffer, report_errors, run_pcmd, \
+    distribute_files, DaosTestError, get_clush_command, run_command
+from ddb_utils import DdbCommand
+from exception_utils import CommandFailure
+
+
+SAMPLE_DKEY = "Sample dkey"
+SAMPLE_AKEY = "Sample akey"
+SAMPLE_DATA = "Sample data"
```

Comment:
Can we put these in the config? Or maybe generate random strings?

Reply:
I updated the code to generate random string for these.